samples/datalimits.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Box Plot Chart</title>
+    <script src="../node_modules/chart.js/dist/Chart.bundle.js"></script>
+    <script src="../build/Chart.BoxPlot.js" type="text/javascript"></script>
+		<script src="./utils.js"></script>
+		<script src="https://d3js.org/d3-random.v1.min.js"></script>
+    <style>
+    canvas {
+        -moz-user-select: none;
+        -webkit-user-select: none;
+        -ms-user-select: none;
+    }
+    </style>
+</head>
+
+<body>
+    <div id="container" style="width: 75%;">
+        <canvas id="canvas"></canvas>
+    </div>
+    <button id="limitMinMax">Limit to min/max</button>
+    <button id="limitWhiskers">Limit to whiskers</button>
+    <script>
+        const samples = this.Samples.utils;
+				var color = Chart.helpers.color;
+				var b = d3.randomNormal();
+				var random = (min, max) => () => (b() * ((max || 1) - (min || 0))) + (min || 0);
+				// var random = d3.randomUniform;
+				var boxplotData = {
+            labels: samples.months({count: 7}),
+            datasets: [{
+                label: 'Dataset 1',
+                backgroundColor: color(window.chartColors.red).alpha(0.5).rgbString(),
+                borderColor: window.chartColors.red,
+                borderWidth: 1,
+                data: samples.boxplots({count: 7, random: random}),
```

Comment:
use shorthand assignment: `{count: 7, random}`

Reply:
do we wanna use full ES6 syntax here?

Reply:
we do not necessarily have to